src/perform_step/low_order.jl

```diff
@@ -98,10 +98,9 @@ end
     integrator.g(gtmp2, tmp, p, t+dt)
 
     if is_diagonal_noise(integrator.sol.prob)
-        @.. nrtmp=(1/2)*W.dW*(gtmp1+gtmp2)
+        @.. nrtmp = (0.5) * W.dW * (gtmp1 + gtmp2)
```

Comment:
```suggestion
        @.. nrtmp = W.dW * (gtmp1 + gtmp2) / 2
```

Otherwise this is tied to Float64

Reply:
Changed to `/2` so it doesn't force `Float64`. Thanks!